Tests/Editor/ProjectAuditorTests.cs

```diff
@@ -106,5 +106,32 @@ public void ProjectAuditor_Params_CallbacksAreInvoked()
             Assert.AreEqual(1, numModules);
             Assert.NotNull(projectReport);
         }
+
+        [Test]
+        public void ProjectAuditor_Report_IsUpdated()
+        {
+            var projectAuditor = new Unity.ProjectAuditor.Editor.ProjectAuditor(m_Config);
+            var report = projectAuditor.Audit(new ProjectAuditorParams
+            {
+                categories = new[] { IssueCategory.ProjectSetting}
+            });
+
+            Assert.True(report.HasCategory(IssueCategory.ProjectSetting));
+            Assert.Positive(report.GetIssues(IssueCategory.ProjectSetting).Count);
```

Comment:
It seems that here we assume that there are always settings issues, right?

To be sure about the unit test some rather simple ProjectSettings issue could be "provoked" here I guess.

Reply:
> It seems that here we assume that there are always settings issues, right?
> 
> To be sure about the unit test some rather simple ProjectSettings issue could be "provoked" here I guess.

You are absolutely right. I'll add that.